src/main/java/org/aion/wallet/ui/components/ReceiveController.java

```diff
@@ -41,6 +47,13 @@ private void handleAccountChanged(final AccountEvent event) {
         if (AccountEvent.Type.CHANGED.equals(event.getType())) {
             account = event.getAccount();
             accountAddress.setText(account.getPublicAddress());
+
+            Image image = SwingFXUtils.toFXImage(account.getQrCode(), null);
+            qrCode.setImage(image);
+            qrCode.setFitHeight(0);
+            qrCode.setFitWidth(0);
+            qrCode.setSmooth(true);
```

Comment:
THe following settings are actual defaults, so they can be removed:
`qrCode.setFitHeight(0);`
`qrCode.setFitWidth(0);`
`qrCode.setSmooth(true);`